spec/features/parental_consent_refused_spec.rb

```diff
@@ -152,12 +152,13 @@ def and_the_action_in_the_vaccination_session_is_to_check_refusal
     end
     click_on "Pilot School"
 
-    click_on "Outcome"
-
-    expect(page).to have_content("Could not vaccinate")
-    choose "Could not vaccinate"
-    click_on "Update results"
-
-    expect(page).to have_content(@child.full_name)
+    # TODO: Check in programme children tab
```

Comment:
What's the plan here?

Reply:
Follow up PR coming later today that builds the programme children tab.